examples/fieldLevelValidation/src/FieldLevelValidationForm.js

```diff
@@ -58,7 +59,7 @@ const FieldLevelValidationForm = props => {
         type="text"
         component={renderField}
         label="Username"
-        validate={[required, maxLength15, minLength2]}
+        validate={[required, maxLength15, minLength2, minLength4]}
```

Comment:
This doesn't make any sense. Perhaps it slipped in. Remove this and I'll merge the dep upgrades.

Reply:
Oh, I'm sorry. That was my experiment for having multiple errors.
I've just removed this.